tests/casp/filesystem_ro.pm

```diff
@@ -16,12 +16,8 @@ use testapi;
 
 sub run() {
     assert_script_run "! touch /should_fail";
-
-    # /etc is ro now but may be changed later
-    assert_script_run "! touch /etc/should_fail";
-
+    assert_script_run "touch /etc/should_succeed";
```

Comment:
Hm, the changes are pretty straight forward, but please take in mind that you should have `root privileges` in order to have `write` access into the `/etc/` directory. Otherwise your test will fail like this:

```
$:~> touch /etc/should_succeed
touch: cannot touch '/etc/should_succeed': Permission denied
```
In order to log in as root, you can use `select_console "root-console";` or at least make sure that you are `root` and not a normal user before issuing this command.